modules/PyEvJosev/module.py

```diff
@@ -0,0 +1,113 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+import asyncio
+import sys
+from pathlib import Path
+import threading
+
+from everest.framework import Module, RuntimeSession, log
+
+# fmt: off
+JOSEV_WORK_DIR = Path(__file__).parent / '../../3rd_party/josev'
+sys.path.append(JOSEV_WORK_DIR.as_posix())
+
+from iso15118.evcc import EVCCHandler
+from iso15118.evcc.controller.simulator import SimEVController
+from iso15118.evcc.evcc_config import EVCCConfig
+from iso15118.evcc.everest import context as JOSEV_CONTEXT
+from iso15118.shared.exificient_exi_codec import ExificientEXICodec
+from iso15118.shared.settings import set_PKI_PATH
```

Comment:
I would prefer having this solved differently / we need to come up with a best practice how external python modules are distributed to the system
- either by copying them directly
- or by pip
- wheels?